docs/source/rest-api.rst

```diff
@@ -1797,6 +1797,415 @@ Response Structure
 | artifact_uri | ``STRING`` | URI corresponding to where artifacts for this model version are stored. |
 +--------------+------------+-------------------------------------------------------------------------+
 
+===========================
```

Comment:
Is it possible to isolate these endpoints into a different page?

Reply:
Feel free to create a folder for auth and put relevant files there.

Reply:
sure, sounds good!